src/anomalib/engine/engine.py

```diff
@@ -897,22 +897,25 @@ def export(
         CLI Usage:
             1. To export as a torch ``.pt`` file you can run the following command.
                 ```python
-                anomalib export --model Padim --export_mode TORCH --data MVTec
+                anomalib export --model Padim --export_mode TORCH --ckpt_path <PATH_TO_CHECKPOINT>
```

Comment:
Do we still need to provide `--export_mode TORCH` or does `torch` also work?

Reply:
torch works as well

Reply:
my only concern is that there is consistency with the arguments. For example model starts with a upper-case, export_mode is all upper-case. As long as we support all of the combinations, that's fine. Otherwise, we might need to re-visit this in the future.